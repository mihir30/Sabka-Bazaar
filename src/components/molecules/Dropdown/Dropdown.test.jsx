import React from 'react';
import renderer from 'react-test-renderer';
import DropDown from './Dropdown';

it ("Checking test for Dropdown", () => {
    const tree = renderer.create(<DropDown />).toJSON();
    expect(tree).toMatchSnapshot();
});