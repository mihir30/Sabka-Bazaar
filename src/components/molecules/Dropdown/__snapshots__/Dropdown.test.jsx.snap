// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking test for Dropdown 1`] = `
<select
  className="category-dropdown-content"
>
  <option
    value="Default"
  >
    All Products
  </option>
</select>
`;


